### PR TITLE
Added azure-devops topic

### PR DESCRIPTION
**Pull request by brunobritorj**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> The azure-devops topic has more than 1.3k repositories linked but so far it wasn't a curated topic.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

File: topics/azure-devops/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: Microsoft
+display_name: Azure DevOps
+logo: azure-devops.png
+released: 2005, formerly Team Foundation Server (TFS) and Azure DevOps Server
```
Comment (kenyonj) [inline topics/azure-devops/index.md:5]:
```suggestion
released: 2005
```

File: topics/azure-devops/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+short_description: Azure DevOps is a Microsoft product that provides version control, reporting, requirements/project management, and automated builds/tests/release capabilities.
```
Comment (kenyonj) [inline topics/azure-devops/index.md:6]: This needs to be shortened a bit

**Review by kenyonj — CHANGES_REQUESTED** [review]
Looks like there are several failures that need to be addressed. Please take a look at the CI results for more information.

File: topics/azure-devops/index.md (hunk 2 above)
Comment (brunobritorj) [reply topics/azure-devops/index.md]: Done

**Comment by brunobritorj** [comment]
Hi Justin,
Sure, I fixed them.
Thanks!

**Review by kenyonj — APPROVED** [review]

**Review by kenyonj — CHANGES_REQUESTED** [review]
Still a failure around the image dimensions.

**Comment by brunobritorj** [comment]
> Still a failure around the image dimensions.

Fixed. 288x288px with transparent background.

**Review by kenyonj — APPROVED** [review]

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
@brunobritorj Please rebase this on top of `master` or merge in `master`, this has merge conflicts that need resolved. Thanks!

**Comment by brunobritorj** [comment]
Thanks @MikeMcQuaid !

**Comment by MikeMcQuaid** [comment]
Thanks @brunobritorj!

**Pull request merged**